functests/4_latency/latency.go

```diff
@@ -185,6 +195,7 @@ var _ = Describe("[performance] Latency Test", func() {
 			// verify the maximum latency only when it requested, because this value can be very different
 			// on different systems
 			if maximumLatency == -1 {
+				klog.Info("no maximum latency value provided, skips the latency value check")
```

Comment:
what the benefits of this approach over `Skip` are, again?

Reply:
Because even if we are not evaluating the latency values, we still need to make sure the binaries and mechanism are functions properly - which is the main purpose of the test though.
If we'll skip the whole test we won't have any indication, not even if the binary run is good.

Reply:
I think I'm missing something.
With the current approach we go this far, then we return. And the test is accounted as succeeded. We will later need to dig in the logs to learn that it was actually skipped.
If we skip here (instead of return), we can clearly see the test was skipped, and the message is presented to us.

Reply:
Probably @fromanirh is right.
We drop some specific skip message, something like `skip buckets latency check`, and if the test will fail before you still will see the failure.